### ISSUE TITLE
aria-live elements need to be empty or invisible on sim start up

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#159]
In a user test 7/20, the user was able to find the 'aria-live' alert section above the simulation title, which was unexpected and confusing.

This is because the alert had content on sim-load.  The alert should be empty when the sim is initially loaded.

We should also test hiding these elements, that would be ideal.  But I am not sure if AT will still anounce aria-live updates from elements which are hidden.

**Comment by jessegreenberg** [comment]
Above commit empties the live textContent on startup by attaching the model to the element with a lazyLink.
